pkg/common/helpers.go

```diff
@@ -49,3 +64,67 @@ func IsExpired(startTime int64, ttlTime int64) bool {
 	}
 	return false
 }
+
+// NewWFClient -- declare new workflow client
+func NewWFClient(cfg *rest.Config) wfv1versioned.Interface {
+	cli, err := wfv1versioned.NewForConfig(cfg)
+	if err != nil {
+		panic(err)
+	}
+	return cli
+}
+
+// NewAddonClient - declare new addon client
+func NewAddonClient(cfg *rest.Config) addonv1versioned.Interface {
+	cli, err := addonv1versioned.NewForConfig(cfg)
+	if err != nil {
+		panic(err)
+	}
+	return cli
+}
+
+// NewK8sClient defines kubernetes client
+func NewK8sClient(kubeconfigPath string) (kubernetes.Interface, error) {
```

Comment:
We don't need or want this in our code, again `controller-runtime` and manager are providing the client

Reply:
Yeah, this is a brilliant idea. re-use the controller-runtime function would work. thanks.

Reply:
f91a53ffc8d59d0bc3891a8d494fa2ed5eafb406